src/components/cards/Cards.tsx

```diff
@@ -19,4 +21,6 @@ export const Cards: FC<PropsCardsType> = ({ cards }) => (
       />
     ))}
   </div>
+        :
```

Comment:
Fix formatting, please

Reply:
https://github.com/gitkoshelew/CC.ui/pull/54/commits/933e4da7d9c6a8e34aff28fda08bde1f0c7aac74
fix formatting as you wrote earlier